package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "figci-server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www",
+    "dev": "nodemon ./bin/www",
```

```suggestion
    "dev": "NODE_ENV=development nodemon ./bin/www",
```
node.js는 변경사항이 생겨도 `auto-reload`가 되지 않아서 직접 서버를 다시 실행시켜 주어야지 변경사항이 반영이 되는데,
nodemon은 `auto-reload`를 지원하기 때문에 개발을 하는데 많은 간편함을 줍니다!

물론 따로 NODE_ENV를 명시하지 않으면 개발환경으로 실행이 되지만, 위 코드처럼 수정하여 명시적으로 작성해주시면 더 좋지 않을까 생각합니다!

https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production

https://github.com/remy/nodemon#:~:text=nodemon%20is%20a%20tool%20that%20helps%20develop%20Node.js%20based%20applications%20by%20automatically%20restarting%20the%20node%20application%20when%20file%20changes%20in%20the%20directory%20are%20detected.

---

Node.js에서 개발 환경을 명시해준다는 게 제안해주신 코드가 더 좋은 것 같습니다. 
코드에 반영하겠습니다!

---

반영했습니다. 확인 부탁드립니다.